android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -215,7 +273,21 @@ public void requestLocationPermissions(PluginCall call) {
 
     @PluginMethod()
     public void getLocation(final PluginCall call) throws JSONException {
-        Radar.getLocation(new Radar.RadarLocationCallback() {
+        String desiredAccuracy = call.getString("desiredAccuracy");
+        RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy accuracyLevel = RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy.MEDIUM;
+        String accuracy = desiredAccuracy != null ? desiredAccuracy.toLowerCase()  : "medium";
+
+        if (accuracy.equals("low")) {
+            accuracyLevel = RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy.LOW;
+        } else if (accuracy.equals("medium")) {
+            accuracyLevel = RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy.MEDIUM;
+        } else if (accuracy.equals("high")) {
+            accuracyLevel = RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy.HIGH;
+        } else {
+            call.reject(Radar.RadarStatus.ERROR_BAD_REQUEST.toString());
```

Comment:
Let's have the error message be:
`"invalid desiredAccuracy: " + desiredAccuracy`